service-worker/src/job/news_archive_job.go

```diff
@@ -39,64 +39,66 @@ func NewsArchiveJob(conn *utils.Conn, cfg *config.Config) {
 		newsIDs = append(newsIDs, id)
 	}
 
-	// Archive news
-	updateQuery := fmt.Sprintf(`UPDATE news SET is_live=0, status='ARCHIVED' WHERE id IN (%s)`, strings.Join(newsIDs, ","))
-	res, err := conn.Mysql.Exec(updateQuery)
-	if err != nil {
-		logrus.Error(err)
-	}
+	if len(newsIDs) > 0 {
```

Comment:
```suggestion
	if len(newsIDs) == 0 {
	  return
	}
```

I personally write like this, because better code does not code complexity

Reply:
done here mate https://github.com/jabardigitalservice/portal-jabar-services/pull/370/commits/8c5e664e28e18c2259be5c6b05f477dffda057b4 thanks!